pkg/apis/tensorflow/helper/helpers.go

```diff
@@ -110,10 +110,12 @@ func Cleanup(c *tfv1.TFJobSpec) {
 	// We should have default container images so user doesn't have to provide these.
 }
 
+// CRDName : method returns custom resource definition kind and group.
```

Comment:
nit: Go style prefers complete sentences so here it should be
"CRDName returns the custom resource definition name which is a combination of the kind and group."

Please make this change everywhere.

https://golang.org/doc/effective_go.html#commentary